File: Iniciante/PHP/exercise_1035.php
<?php

$valores = readline("Insira 4 valores: ");

list($valorA, $valorB, $valorC, $valorD) = sscanf($valores, "%d %d %d %d");

if ($valorB>$valorC && $valorD>$valorA && (($valorC+$valorD) > ($valorA+$valorB))
    && $valorD >= 0 && $valorC >= 0 && ($valorA%2 === 0)) {
    print_r("Valores aceitos\n");
} else {
    print_r("Valores nao aceitos\n");
}

//Concluído!

File: Iniciante/Python/exercises from 1000 to 1099/exercise_1035.py
valorA, valorB, valorC, valorD = [int(x) for x in input().split()]

isBBiggerThanC = valorB > valorC
isDBiggerThanA = valorD > valorA
isDPlusCBiggerThanAPlusB = (valorD + valorC) > (valorB + valorA)
isCAndDPositive = valorC >= 0 and valorD >= 0
isAPair = valorA%2 == 0

if isBBiggerThanC and isDBiggerThanA and isDPlusCBiggerThanAPlusB and isCAndDPositive and isAPair:
    print('Valores aceitos')
else:
    print('Valores nao aceitos')